db/migrate/20230713182204_create_credential_adoption_email_records.rb

```diff
@@ -0,0 +1,12 @@
+class CreateCredentialAdoptionEmailRecords < ActiveRecord::Migration[6.1]
+  def change
+    create_table :credential_adoption_email_records do |t|
+      t.string :veteran_icn, null: false
+      t.string :veteran_email_address, null: false
```

Comment:
I'd recommend dropping `veteran_` from these column names. Just to keep it constant with other tables.

Also, this should have an index on `icn` and/or `email` - depending on how you are going to be querying this.

Reply:
Hey, @rileyanderson !  

Melissa also brought up the naming convention, so I'll make that change now.

I went back and forth on adding an index on one or both columns.  We anticipate having less than 30,000 records total created for in this database, but we would be checking the table for recent records tied to a particular email in the future so I could see that making use of the query performance gains of having an index.  

If you think the benefit is worth it I can add those indexes in when I do the renaming of the columns here shortly :)

Reply:
changes made!